ui/component/header/view.jsx

```diff
@@ -164,6 +164,7 @@ const Header = (props: Props) => {
                 <Menu>
                   <MenuButton
                     aria-label={__('Publish a file, or create a channel')}
+                    title={__('Publish a file, or create a channel')}
```

Comment:
I think the tooltip needs to be shorter.

Reply:
I think that sentence is fine

Reply:
Isn't there a possibility that it hides other text or gets cut?

Reply:
I think it would look better if it is made "Publish Your Content!" or "Publish Freedom" or something like that. Or just "Publish Content".

This is still my personal opinion. I don't like long tooltips, they look a bit messy to me.

Reply:
Nope, they sit on top of all the other text, so it won't get hidden.